examples/price-calculation/README.md

```diff
@@ -0,0 +1,60 @@
+## Overview
+
+This sample demonstrates the use of a connector generated by the Ballerina MI SDK to perform price related calculation of an order. 
+In this example, the resource `price-calculation` accepts an XML payload containing item details and calculates the total invoice value.
+This calculation is done by a function implemented in Ballerina, which processes an input payload of the following format:
+
+```xml
+<items>
+    <item>
+        <name>{item-name}</name>
+        <price>{price}</price>
+    </item>
+    ...
+</items>
+```
+
+This would output a result in the following format.
+
+```xml
+<total xmlns="http://ws.apache.org/ns/synapse">{total}</total>
+```
+
+## Steps to Invoke the Sample
+
+Follow these steps to invoke the sample using the connector:
+
+1. The `bal-price-calculation` folder contains the ballerina code for the connector. Invoke the following command to generate the connector:
```

Comment:
So we need to navigate there, right? Shall we mention that explicitly?

Reply:
We don't have to navigate to that folder, we can provide the path to that folder.  Navigating to that is not working now due to a bug. https://github.com/ballerina-platform/ballerina-library/issues/7370. I'll leave this as it is then?

Reply:
Ah yes, missed the argument. This is okay.